### ISSUE TITLE
Tasks, Subtasks and Descriptions are not or only partially saving

### ISSUE THREAD

**Issue by new-nextcloud-tasks-user** [nextcloud/tasks#67]
### Steps to reproduce
Create a task, subtask or description/notes to a task or set the priority

### Expected behaviour
Saving changes

### Actual behaviour
Almost everytime changes are lost after reloading the page. Sometimes changes are saved partially.

### Server configuration
**Operating system**: FreeBSD

**Web server:** Apache

**Database:** SQL

**PHP version:** 7

**Nextcloud version:** (see Nextcloud admin page) 11.0.2

**Tasks version:** (see Nextcloud apps page) 0.9.5

**Updated from an older Nextcloud or fresh install:** fresh install

**Signing status:**

```
No errors have been found.
```

**List of activated apps:**

```
...
```

**The content of config/config.php:**

```
...
```

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

### Client configuration
**Browser:** Vivaldi

**Operating system:** Windows 10

**CalDAV-clients:** DAVDroid and Thunderbird

#### Web server error log
```
...
```

#### Nextcloud log (data/nextcloud.log)
```
...
```

#### Browser log
```
Deprecation warning: use moment.updateLocale(localeName, config) to change an existing locale. moment.defineLocale(localeName, config) should only be used for creating a new locale See http://momentjs.com/guides/#/warnings/define-locale/ for more info.
warn @ moment-with-locales.js?v=b7ecb20…:282
deprecateSimple @ moment-with-locales.js?v=b7ecb20…:323
defineLocale @ moment-with-locales.js?v=b7ecb20…:1833
locale_locales__getSetGlobalLocale @ moment-with-locales.js?v=b7ecb20…:1816
(anonymous) @ app.js?v=b7ecb20…:152
invoke @ angular.min.js?v=b7ecb20…:41
(anonymous) @ angular.min.js?v=b7ecb20…:43
q @ angular.min.js?v=b7ecb20…:7
bb @ angular.min.js?v=b7ecb20…:43
c @ angular.min.js?v=b7ecb20…:21
yc @ angular.min.js?v=b7ecb20…:21
ee @ angular.min.js?v=b7ecb20…:20
(anonymous) @ angular.min.js?v=b7ecb20…:313
j @ jquery.js:3099
fireWith @ jquery.js:3211
ready @ jquery.js:3417
I @ jquery.js:3433
tasks:1 GET /index.php/apps/theming/favicon/tasks?v=3 500 ()
DevTools failed to parse SourceMap: /index.php/apps/tasks/dist/purify.min.js.map
```

**Comment by Aninstance** [comment]
Same issue here, running Arch Linux and Chromium. Saving tasks is very hit 'n miss. They appear to be saved correctly, but following a refresh or new browser session they're gone.

**Comment by treiher** [comment]
I observed a similar behavior. I guess it is related to an interruption of the network connection. The issue usually occurs after reconnecting to a network after being disconnected for some time, or a wake up from a suspend. In these cases I'm able to change tasks without any error reported and all changes are shown correctly. But a look in the network log shows that all requests get a response with `Status Code: 401 Unauthorized`. Thus all changes are done in the front end, but not persistently stored. After reloading the page all changes are lost, but I'm still logged in and adding/changing tasks work again as expected.

I observed the same issue for the calendar app. I would expect that at least a error is reported to the user if it fails to permanently store a task (or event).
